storage/metadata_storage.go

```diff
@@ -212,7 +212,8 @@ type metadataStorageImplementation interface {
 	// Set stores the value for the key and updates it if it already exists
 	Set(key string, value string) error
 	// Get returns the value for the key
-	Get(key string, opts ...query.Query) (string, error)
+	Get(key string) (string, error)
+	GetWithOpts(key string, opts ...query.Query) (string, error)
```

Comment:
I only added this to fix a merge conflict, you can merge them back together under Get